prisma/schema.prisma

```diff
@@ -129,8 +129,10 @@ model LearningJourney {
   updatedAt DateTime @default(now()) @updatedAt @map("updated_at") @db.Timestamptz(3)
 
   // Domain-Specific Fields.
-  isCompleted    Boolean @default(false) @map("is_completed")
-  lastCheckpoint Decimal @map("last_checkpoint")
+  isCompleted     Boolean @default(false) @map("is_completed")
+  lastCheckpoint  Decimal @map("last_checkpoint")
+  isQuizAttempted Boolean? @map("is_quiz_attempted")
+  isQuizPassed    Boolean? @map("is_quiz_passed")
```

Comment:
Should these two fields be nullable or should it always default to false instead?

Reply:
I made it nullable because non-mandatory learning bites doesn't use these fields.